BootloaderCorePkg/Library/PciEnumerationLib/PciEnumerationLib.c

```diff
@@ -100,8 +103,11 @@ PciDevicePresent (
   if (PciExpressRead16 (Address) == 0xFFFF) {
     return EFI_NOT_FOUND;
   } else {
-    CopyMem (Pci, (VOID *) ((UINTN)PcdGet64 (PcdPciExpressBaseAddress) + \
-                            PCI_EXPRESS_LIB_ADDRESS (Bus, Device, Func, 0)), sizeof (PCI_TYPE00));
+    Src = (UINT32 *) ((UINTN)PcdGet64 (PcdPciExpressBaseAddress) + PCI_EXPRESS_LIB_ADDRESS (Bus, Device, Func, 0));
```

Comment:
Maybe we need to add comments to explain why we did not use MemCopy().

Reply:
Added a comment in the code and mentioned in commit message.